lib/utils/stability_score.dart

```diff
@@ -0,0 +1,31 @@
+import '../types/session_status.dart';
+import '../types/sessions.dart';
+
+extension StabilityScore on Sessions {
+  double stabilityScore() {
+    int healthy;
+    int errored;
+    int abnormal;
+    int crashed;
+    for (final group in groups) {
+      if (group?.by?.sessionStatus == SessionStatus.healthy) {
+        healthy = group?.totals?.sumSession;
+      } else if (group?.by?.sessionStatus == SessionStatus.errored) {
+        errored = group?.totals?.sumSession;
+      } else if (group?.by?.sessionStatus == SessionStatus.abnormal) {
+        abnormal = group?.totals?.sumSession;
+      } else if (group?.by?.sessionStatus == SessionStatus.crashed) {
+        crashed = group?.totals?.sumSession;
+      }
+    }
+    if (healthy == null || errored == null || abnormal == null || crashed == null) {
+      return null;
+    }
+    final sum = healthy + abnormal + crashed + errored;
+    if (sum == 0) {
+      return 100;
```

Comment:
If the sum is 0 should we assume that there are no sessions in the time period and therefore stability is not 100% but rather `-`?